src/ui-presets/error.js

```diff
@@ -0,0 +1,19 @@
+//@flow
+import style from '../styles/style.scss';
+import {h} from 'preact';
+import ErrorOverlay from '../components/error-overlay';
+
+/**
+ * Error ui interface
+ *
+ * @export
+ * @param {*} props component props
+ * @returns {React$Element} player ui tree
+ */
+export default function errorUI(props: any): React$Element<any> {
+  return (
+    <div className={style.playbackGuiWWrapper}>
+      <ErrorOverlay player={props.player}/>
```

Comment:
Why all the elements are in the OverlayError components? This is very hard to reuse like this.
As I see it the overlay should only define the overlay itself (look on the overlay-play for reference) and the other elements should be map to react components underneath